assemblies/assembly-customizing-the-global-header.adoc

```diff
@@ -0,0 +1,8 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: customizing-the-global-header
+[id="assembly-customizing-red-hat-developer-hub-global-header"]
+= Customizing the {product} global header
+
+As an administrator, you can enable or disable the {product} global header to create a consistent and flexible navigation bar across your {product-custom-resource-type} instance. You can extend the global header with additional buttons and customize the order and position of icons and features. By default, the {product} global header includes a search field, create, support and notification buttons, and a user profile drop-down.
+
+include::modules/customizing-the-global-header/proc-customizing-rhdh-global-header.adoc[leveloffset=+1]
```

Comment:
This is an assembly with only one procedure. For minimalism, consider removing the assembly, and moving the assembly introduction to the procedure.